src/main/java/utility/Main.java

```diff
@@ -11,13 +11,13 @@
 	 * @param args
 	 */
 	public static void main(String[] args) {
-		// If we have a command line arg run the GUI environment
-		if (args.length > 0) {
+		//If we have a command line arg run the GUI environment
+		//if (args.length > 0) {
 			runGUI(args);
-		} else {
```

Comment:
Was this intended?

Reply:
main shouldn't have had things commented out, it was only like that to be able to text GUI. should have been taken out before a PR was made